geopandas/tests/test_plotting.py

```diff
@@ -1748,8 +1741,6 @@ def compare_figures(self, kind, fig_test, fig_ref, kwargs):
 
     _pandas_kinds = []
 
-    _pandas_kinds = GeoplotAccessor._pandas_kinds
```

Comment:
Is this right? If anything I would expect the line above to be removed?

Reply:
Good catch. It seems that ruff's `--fix` option is not perfect here.